docs/guides/integration-aws-entity-resolution.md

```diff
@@ -150,14 +150,31 @@ To create the matching workflow, first sign in to the AWS Management console, op
    - **Description**: An optional description for the matching workflow.
    - **Data input**: The source for your data in AWS. From the drop-down list, select an AWS Glue database, then the AWS Glue table, and then the schema mapping. You can specify more than one data input. In the following example, email is the only type of data input:
 
-     |UniqueId|Name|Email|Date of Birth|
-     |--------|----|-----|-----------|
-     |0001|Test 1|test1@uidapi.com|1/1/90|
-     |0002|Test 2|test2@gmail.com|1/2/78|
+     | UniqueId | Name   | Email            | Date of Birth |
+     |----------|--------|------------------|---------------|
+     | 0001     | Test 1 | test1@uidapi.com | 1/1/90        |
+     | 0002     | Test 2 | test2@gmail.com  | 1/2/78        |
 
      NOTE: If you have both email and phone numbers in the same record, the workflow duplicates each record in the output. If you don't want this, the best approach is to create a separate workflow for each. See details earlier in this section.
 
-   - **Service access**: Grant specific permission to Entity Resolution to access the specified data in AWS Glue, using an existing or new service role. If the input data is encrypted, you must also specify the AWS Key Management Service (KMS) key for decryption.
+   - **Service access**: Grant permissions to Entity Resolution to access the specified data in AWS Glue and to call AWS Data Exchange on your behalf. We highly recommend selecting "Create and use a new service role" and let the workflow create a new role with all the required permissions. If you are using an existing service role then make sure it has permission to call AWS Data Exchange by adding the following permission
```

Comment:
Suggest a couple of tweaks:

Service access: Grant permissions to Entity Resolution to access the specified data in AWS Glue and to call AWS Data Exchange on your behalf. We highly recommend that you select "Create and use a new service role" and let the workflow create a new role with all the required permissions. If you're using an existing service role, make sure that it is authorized to call AWS Data Exchange by adding the following permission: